packages/codefixes/src/codefixes.ts

```diff
@@ -14,4 +15,7 @@ export const codefixes = new CodeFixesProvider([
     4082: new AddMissingExportCodeFix(),
   }),
   new TypescriptCodeFixCollection(),
+  new BaseCodeFixCollection({
+    7006: new AddMissingTypesBasedOnInheritanceCodeFix(),
+  }),
```

Comment:
Is there a reason to add this collection here? Why wouldn't we have it in the first collection?

Reply:
Codefixes are applied in the order defined there.
I want to try this one after TypeScrupt ones were tried to apply.